src/EdgeGPT.py

```diff
@@ -96,7 +95,7 @@ class ConversationStyle(Enum):
 
 
 CONVERSATION_STYLE_TYPE = Optional[
-    Union[ConversationStyle, Literal["creative", "balanced", "precise"]]
+    ConversationStyle | Literal["creative", "balanced", "precise"]
```

Comment:
Does this work in Python 3.8?

Reply:
I just checked—It looks like you can't. I will revert this.